webcommon/libs.nashorn/licenseinfo.xml

```diff
@@ -0,0 +1,129 @@
+        <file>src/com/oracle/js/parser/ir/ImportNode.java</file>
+        <file>src/com/oracle/js/parser/ir/ImportSpecifierNode.java</file>
+        <file>src/com/oracle/js/parser/ir/IndexNode.java</file>
+        <file>src/com/oracle/js/parser/ir/JoinPredecessorExpression.java</file>
+        <file>src/com/oracle/js/parser/ir/JsxAttributeNode.java</file>
+        <file>src/com/oracle/js/parser/ir/JsxElementNode.java</file>
+        <file>src/com/oracle/js/parser/ir/JumpStatement.java</file>
+        <file>src/com/oracle/js/parser/ir/Label.java</file>
+        <file>src/com/oracle/js/parser/ir/LabelNode.java</file>
+        <file>src/com/oracle/js/parser/ir/Labels.java</file>
+        <file>src/com/oracle/js/parser/ir/LexicalContext.java</file>
+        <file>src/com/oracle/js/parser/ir/LexicalContextExpression.java</file>
+        <file>src/com/oracle/js/parser/ir/LexicalContextNode.java</file>
+        <file>src/com/oracle/js/parser/ir/LexicalContextStatement.java</file>
+        <file>src/com/oracle/js/parser/ir/LiteralNode.java</file>
+        <file>src/com/oracle/js/parser/ir/LoopNode.java</file>
+        <file>src/com/oracle/js/parser/ir/Module.java</file>
+        <file>src/com/oracle/js/parser/ir/NameSpaceImportNode.java</file>
+        <file>src/com/oracle/js/parser/ir/NamedImportsNode.java</file>
+        <file>src/com/oracle/js/parser/ir/Node.java</file>
+        <file>src/com/oracle/js/parser/ir/ObjectNode.java</file>
+        <file>src/com/oracle/js/parser/ir/PropertyKey.java</file>
+        <file>src/com/oracle/js/parser/ir/PropertyNode.java</file>
+        <file>src/com/oracle/js/parser/ir/ReturnNode.java</file>
+        <file>src/com/oracle/js/parser/ir/RuntimeNode.java</file>
+        <file>src/com/oracle/js/parser/ir/Statement.java</file>
+        <file>src/com/oracle/js/parser/ir/SwitchNode.java</file>
+        <file>src/com/oracle/js/parser/ir/Symbol.java</file>
+        <file>src/com/oracle/js/parser/ir/Terminal.java</file>
+        <file>src/com/oracle/js/parser/ir/TernaryNode.java</file>
+        <file>src/com/oracle/js/parser/ir/ThrowNode.java</file>
+        <file>src/com/oracle/js/parser/ir/TryNode.java</file>
+        <file>src/com/oracle/js/parser/ir/UnaryNode.java</file>
+        <file>src/com/oracle/js/parser/ir/VarNode.java</file>
+        <file>src/com/oracle/js/parser/ir/WhileNode.java</file>
+        <file>src/com/oracle/js/parser/ir/WithNode.java</file>
+        <file>src/com/oracle/js/parser/ir/visitor/NodeVisitor.java</file>
+        <file>src/com/oracle/js/parser/ir/visitor/TranslatorNodeVisitor.java</file>
+        <file>src/com/oracle/js/parser/package-info.java</file>
+        <file>src/com/oracle/js/parser/resources/Messages.properties</file>
+        <file>test/unit/src/com/oracle/js/parser/DumpingVisitor.java</file>
+        <file>test/unit/src/com/oracle/js/parser/ManualTest.java</file>
+        <file>test/unit/src/com/oracle/js/parser/ParserTest.java</file>
+        <license ref="UPL" />
```

Comment:
Do we want to use [UPL-Apache-2.0](https://github.com/apache/netbeans/blob/master/nbbuild/licenses/UPL-Apache-2.0) instead? 

Or we may even want to create a new UPL-Oracle license:

- The "UPL" license has a generic copyright information.
- UPL-Apache-2.0 includes Oracle's copyright, but for years 2016-2020.
- We are importing files with Oracle copyright for 2010-2018, though.